src/NuGet.Core/NuGet.Packaging/Signing/Utility/SignatureUtility.cs

```diff
@@ -16,14 +15,17 @@ namespace NuGet.Packaging.Signing
 {
     public static class SignatureUtility
     {
+
+        //Length of a SHA-1 hash byte array 
+        private const int HashLength = 20;
```

Comment:
Consider adding `Sha1` to the name: 

```suggestion
        private const int Sha1HashLength = 20;
```

Reply:
Yes, and then remove the comment.

Reply:
Thanks for reviewing! Fixed.